datastore/__init__.py

```diff
@@ -251,6 +251,30 @@ async def register_inference_result(
         raise Exception("Unhandled Error")
 
 
+async def new_perfect_inference_feeback(cursor, inference_id, user_id, boxes_id) :
+    """
+    Args:
+        cursor: The cursor object to interact with the database.
+        inference_id (str): id of the inference on which feedback is given
+        user_id (str): id of the user giving a feedback
+        boxes_id (str array): array of id of the objects that are correctly identified
+    """
+    try:
+        for object_id in boxes_id:
```

Comment:
We should check if the boxes exists and return and specific error to let the users know what is wrong.